lightblue/src/main/java/org/esbtools/eventhandler/lightblue/LightblueNotificationRepository.java

```diff
@@ -27,12 +26,7 @@
 import org.esbtools.eventhandler.lightblue.client.LightblueErrors;
 import org.esbtools.eventhandler.lightblue.client.LightblueRequester;
 import org.esbtools.eventhandler.lightblue.client.UpdateRequests;
-import org.esbtools.eventhandler.lightblue.locking.LockNotAvailableException;
 import org.esbtools.eventhandler.lightblue.locking.LockStrategy;
```

Comment:
Now that locking is gone, we can actually remove this import, the field, and associated constructor parameter. This will likely require updates to our internal code which I believe expects to be passing this constructor parameter.